conda.recipe/meta.yaml

```diff
@@ -15,31 +15,40 @@ source:
 requirements:
   build:
```

Comment:
we don't have any dependencies in our setup.py though, we are not building anything